packages/gatsby-plugin-netlify/src/gatsby-node.js

```diff
@@ -52,3 +52,38 @@ exports.onPostBuild = async (
     createRedirects(pluginData, redirects, rewrites),
   ])
 }
+
+const pluginOptionsSchema = function ({ Joi }) {
+  // headers is a specific type used by Netlify: https://www.gatsbyjs.com/plugins/gatsby-plugin-netlify/#headers
+  const headersSchema = Joi.object()
+    .pattern(/^/, Joi.array().items(Joi.string()))
```

Comment:
Maybe we could make this more clear by storing the regex in a variable?

```JS
const MATCH_ALL_KEYS = /^/
.pattern(MATCH_ALL_KEYS, ...
```

Reply:
Yes for sure, let me do this 😊 . Thanks for the the suggestion 🚀